### PR TITLE
Add rx-angular@1.0.4

### PR DESCRIPTION
**Pull request by Piicksarn**
@maruilian11,  would you mind to have a look at the pr?
The PR is referenced to issue  #6040 
The git repo :  https://github.com/Reactive-Extensions/rx.angular.js/tree/v1.0.4

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `37`, Star `429`, Fork `66`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

Thank you! :smile_cat:

### CODE REVIEW

**Comment by maruilian11** [comment]
@PeterDaveHello i think the pr is ok.

**Comment by PeterDaveHello** [comment]
rebase first ....

**Comment by Piicksarn** [comment]
@maruilian11, i have it rebased!
Would you mind to have a look at the pr again?
Thank you!

File: ajax/libs/rx-angular/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "1.0.4",
+  "homepage": "https://github.com/Reactive-Extensions/rx.angular.js",
+  "author": {
+    "name": "Microsoft"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Reactive-Extensions/rx.angular.js.git"
+  },
+  "licenses": "Apache-2.0",
```
Comment (maruilian11) [inline ajax/libs/rx-angular/package.json]: could you modify it to `license`? thanks

**Comment by Piicksarn** [comment]
@maruilian11, i have it rebased!
would you mind to have a look at the pr?
Thank you!

**Comment by maruilian11** [comment]
@Piicksarn could you add `repository`?

**Pull request merged**